.github/workflows/maven.yml

```diff
@@ -1,13 +1,7 @@
 ## Licensed under the terms of http://www.apache.org/licenses/LICENSE-2.0
 
 name: Apache Jena CI
-
-on:
-  push:
-    branches: [ main ]
-#  pull_request:
-#    branches: [ main ]
-
+on: workflow_dispatch
```

Comment:
Doesn't it make every other build, even linux, triggered manually only?

Reply:
Yes. I think there has to be be individual workflow files maven.yml (linux, always run on main?), maven_windows.yml?, maven_macos.yml?, if only some  are to be triggered manually.

Reply:
I think so. At the project I am working on at $work, we have a [separate workflow](https://github.com/cylc/cylc-flow/blob/b5955ab1c2b16cc0a76baaf5826748bb8c6e289f/.github/workflows/1_create_release_pr.yml) for some release automation. Maybe we can have a `maven_windows.yml` or `windows.yml`. The rest of the file looks good after a cursory look :+1: